python/tests/test_generate_gherkin_feature_files.py

```diff
@@ -1,5 +1,7 @@
 from click.testing import CliRunner
+from os import listdir, path
 from typing import Tuple
+from tempfile import TemporaryDirectory
```

Comment:
```suggestion
from tempfile import TemporaryDirectory
from typing import Tuple
```

remember to keep imports alphabetized to be easier to navigate when there is multiple.